.github/copilot-instructions.md

```
@@ -209,6 +230,88 @@ src/
+
+**Common Test Patterns**:
+```csharp
+[Fact]
+public void GenerateCode_ReportsProgress()
+    => mock.Verify(c => c.Progress(It.IsAny<uint>(), It.IsAny<uint>()));
+    
+[SkippableFact(typeof(ProcessLaunchException))]  // Handle network/tool failures
+public void InstallTool_Returns_Path() { ... }
+```
+
+## Validation Scenarios
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)